.github/workflows/qiita-ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
-          python-version: 3.6
+          python-version: '3.9'
```

Comment:
Before this was 3.6 as a float. Just to confirm, should it be in fact a string?

Reply:
Yes, this is fine. Note that in [the documentation](https://github.com/actions/setup-python) it's actually a string, I guess to support non numeric versions.